projects/demo/src/modules/components/input-password/input-password.module.ts

```diff
@@ -25,6 +26,7 @@ import {ExampleTuiInputPasswordComponent} from './input-password.component';
         InheritedDocumentationModule,
         TuiAddonDocModule,
         TuiButtonModule,
+        TuiModeModule,
```

Comment:
There's no need for this module since you are not using the directive here.

Reply:
My fault, I forgot to remove it